<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(164, 208, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_new_book_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 248, 255);</string>
    </property>
    <property name="text">
     <string>Добавить новую книгу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_book_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>400</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 248, 255);</string>
    </property>
    <property name="text">
     <string>Выбрать книгу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quotes_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans Light</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(216, 248, 255);</string>
    </property>
    <property name="text">
     <string>Цитаты и комментарии</string>
    </property>
   </widget>
   <widget class="QLabel" name="prog_name_lbl">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Imprint MT Shadow</family>
      <pointsize>48</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(17, 140, 255);</string>
    </property>
    <property name="text">
     <string>E-book reader</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
